### PR TITLE
Vi 55

### PR DESCRIPTION
**Pull request by mSobonVirtua**

### CODE REVIEW

File: my-project/src/Controller/RegisterController.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * VI-55 Registration
+ *
+ * @category   RegistrationController
+ * @package    Virtua_Registration
```
Comment (astyczen) [inline my-project/src/Controller/RegisterController.php:6]: @mSobonVirtua lets change all packages to Virtua_Registration, also add headers where missing

File: my-project/src/Controller/RegisterController.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+     * @Route("/register", name="userRegister")
+     */
+    public function index(Request $request, UserPasswordEncoderInterface $passwordEncoder, RoleRepository $roleRepository)
+    {
+        $user = new User();
+        $form = $this->createForm(UserType::class, $user);
+        $form->handleRequest($request);
+
+        if($form->isSubmitted() && $form->isValid())
+        {
```
Comment (astyczen) [inline my-project/src/Controller/RegisterController.php:34]: @mSobonVirtua we should add additional valitation for form fields

**Review by astyczen — COMMENTED** [review]
@mSobonVirtua is everything validation correctly on frontend?

File: my-project/src/Controller/RegisterController.php (hunk 2 above)
Comment (mSobonVirtua) [reply my-project/src/Controller/RegisterController.php:34]: @astyczen What do you mean? I tested validation and it working fine.

Comment (astyczen) [reply my-project/src/Controller/RegisterController.php:34]: @mSobonVirtua like trimming, stripping slashes and other safety issues

Comment (mSobonVirtua) [reply my-project/src/Controller/RegisterController.php:34]: @astyczen I read about it and symfony do things like this automatically.  Krzysztofa confirmed me this.

File: my-project/src/Controller/ProductCategoryController.php [hunk 3]
```diff
@@ -120,9 +120,7 @@ public function show(ProductCategory $productCategory): Response
     public function edit(Request $request, ProductCategory $productCategory, FileUploaderService $fileUploader): Response
     {
         $prevMainImage = $productCategory->getMainImage();
-        $form = $this->createForm(ProductCategoryType::class, $productCategory, [
```
Comment (astyczen) [inline my-project/src/Controller/ProductCategoryController.php:123]: this should be done in a separate pr as its not according user registration

**Review by astyczen — APPROVED** [review]
